config.js

```diff
@@ -2,16 +2,17 @@ var IRCConnector = require("./classes/IRCConnector.class");
 var JSONConnector = require("./classes/persistence/JSONConnector.class");
 
 // Sample Slack-thru-IRC config
-/*module.exports = {
+module.exports = {
```

Comment:
May be those modifications should be kept local? I test mostly outside slack so the file will keep bouncing back and forth... let's discuss it... I'll merge for now.